### DISCUSSION TITLE
conversation_id

### DISCUSSION THREAD

**Discussion by hrizal** [acheong08/ChatGPT#822] [Q&A] [answered]
I tried the contents of the config to be used as a parameter when calling this lib

`chatbot = Chatbot(config={"access_token":"blablabla", "conversation_id":sys.argv[2], "parent_id":sys.argv[3]}) 
`

how to use the conversation_id parameter exactly ?
I've tried, but it seems the conversation never connects to the previous one.
thanks

**Comment by MrJack100** [comment, marked as answer]
Use chatbot.ask(PARAMS HERE)
